zebrad/src/components/mempool.rs

```diff
@@ -116,10 +119,15 @@ pub struct Mempool {
     /// Handle to the transaction verifier service.
     /// Used to construct the transaction downloader.
     tx_verifier: TxVerifier,
+
+    /// Sender part of a gossip transactions channel.
+    /// Used to broadcast transaction ids to peers.
+    transaction_sender: watch::Sender<Option<UnminedTxId>>,
 }
 
 impl Mempool {
     #[allow(dead_code)]
+    #[allow(clippy::too_many_arguments)]
```

Comment:
https://github.com/rust-lang/rust-clippy/issues/1576

Reply:
Since we're not using the `_network` argument, you could just remove it.

If a future PR needs the `Network`, we can add it back in then.